.github/workflows/build.yaml

```diff
@@ -45,3 +45,40 @@ jobs:
 
       - name: Run phpstan/phpstan
         run: vendor/bin/phpstan analyse
+
+  coding-standards:
```

Comment:
Other than 

- using a different job identifier
- using a different job name
- running `php-cs-fixer` instead of `phpstan`

there is no difference between this pull request and #28.